src/java.base/share/classes/java/util/stream/StreamOpFlag.java

```diff
@@ -325,12 +325,21 @@
      */
     // 12, 0x01000000
     SHORT_CIRCUIT(12,
-                  set(Type.OP).set(Type.TERMINAL_OP));
+                  set(Type.OP).set(Type.TERMINAL_OP)),
 
+    /**
+     * Characteristic value signifying that an operation may adjust the
+     * total size of the stream.
+     * <p>
+     * An intermediate operation can preserve or inject this value.
+     */
+    // 13, 0x04000000
+    SIZE_ADJUSTING(13,
+                   set(Type.OP));
+    
     // The following 2 flags are currently undefined and a free for any further
```

Comment:
For some reason, the comment says 'The following 2 flags', while we had three, so it was incorrect before but correct now.